plugins/tekton/README.md

```diff
@@ -76,13 +76,13 @@ The Tekton plugin enables you to visualize the `PipelineRun` resources available
     backstage.io/kubernetes-namespace: <RESOURCE_NS>
   ```
 
-- The following annotation is added to the `catalog-info.yaml` file of entity to view the latest `PipelineRun` in the CI/CD tab of the application:
+- The following annotation is added to the `catalog-info.yaml` file of the entity to enable the Tekton related features in Backstage. The value of the annotation identifies the name of the Backstage entity:
 
   ```yaml
   annotations:
     ...
 
-    janus-idp.io/tekton-enabled : 'true'
+    janus-idp.io/tekton : <BACKSTAGE_ENTITY_NAME>
```

Comment:
currently, this annotations just help to detect if Tekton needs to be enabled or not 

can we have something as below

```
janus-idp.io/tekton/porject-slug: <value>
janus-idp.io/tekton/app-name : <value>
```


@christophe-f Let us know your thoughts.

Reply:
It's fine like this.